src/main/java/graphql/servlet/AbstractGraphQLHttpServlet.java

```diff
@@ -46,13 +58,13 @@
  */
 public abstract class AbstractGraphQLHttpServlet extends HttpServlet implements Servlet, GraphQLMBean {
 
-    public static final Logger log = LoggerFactory.getLogger(AbstractGraphQLHttpServlet.class);
+    private static final Logger log = LoggerFactory.getLogger(AbstractGraphQLHttpServlet.class);
 
-    public static final String APPLICATION_JSON_UTF8 = "application/json;charset=UTF-8";
-    public static final String APPLICATION_EVENT_STREAM_UTF8 = "text/event-stream;charset=UTF-8";
-    public static final String APPLICATION_GRAPHQL = "application/graphql";
-    public static final int STATUS_OK = 200;
-    public static final int STATUS_BAD_REQUEST = 400;
+    private static final String APPLICATION_JSON_UTF8 = "application/json;charset=UTF-8";
```

Comment:
just wanted to call this out to make sure we are ok making these private. They were used in other classes but I refactored that away.